src/equations/ideal_glm_mhd_2d.jl

```diff
@@ -279,6 +281,194 @@ end
     return f
 end
 
+"""
+    flux_nonconservative_powell_local_symmetric(u_ll, u_rr,
+                                                orientation::Integer,
+                                                equations::IdealGlmMhdEquations2D)
+
+Non-symmetric two-point flux discretizing the nonconservative (source) term of
+Powell and the Galilean nonconservative term associated with the GLM multiplier
+of the [`IdealGlmMhdEquations2D`](@ref).
+
+This implementation uses a non-conservative term that can be written as the product
+of local and symmetric parts. It is equivalent to the non-conservative flux of Bohm
+et al. (`flux_nonconservative_powell`) for conforming meshes but it yields different
+results on non-conforming meshes(!).
+
+The two functions below, which share the same name, return yield either the local 
```

Comment:
```suggestion
The two other flux functions with the same name return either the local 
```
"yield return" was redundant

Reply:
Thanks! I changed my mind about what verb to use and then left the two :facepalm: